package-lock.json

```diff
@@ -0,0 +1,13659 @@
+                "chokidar": "1.7.0",
+                "connect": "3.5.0",
+                "dev-ip": "1.0.1",
+                "easy-extender": "2.3.2",
+                "eazy-logger": "3.0.2",
+                "emitter-steward": "1.0.0",
+                "fs-extra": "3.0.1",
+                "http-proxy": "1.15.2",
+                "immutable": "3.8.1",
+                "localtunnel": "1.8.3",
+                "micromatch": "2.3.11",
+                "opn": "4.0.2",
+                "portscanner": "2.1.1",
+                "qs": "6.2.1",
+                "resp-modifier": "6.0.2",
+                "rx": "4.1.0",
+                "serve-index": "1.8.0",
+                "serve-static": "1.12.2",
+                "server-destroy": "1.0.1",
+                "socket.io": "1.6.0",
+                "socket.io-client": "1.6.0",
+                "ua-parser-js": "0.7.12",
+                "yargs": "6.4.0"
+            },
+            "dependencies": {
+                "after": {
+                    "version": "0.8.1",
+                    "resolved": "https://registry.npmjs.org/after/-/after-0.8.1.tgz",
+                    "integrity": "sha1-q11PuIP1loFtNRX495HAr0ht1ic=",
+                    "dev": true
+                },
+                "base64id": {
+                    "version": "0.1.0",
+                    "resolved": "https://registry.npmjs.org/base64id/-/base64id-0.1.0.tgz",
+                    "integrity": "sha1-As4P3u4M709ACA4ec+g08LG/zj8=",
+                    "dev": true
+                },
+                "batch": {
+                    "version": "0.5.3",
+                    "resolved": "https://registry.npmjs.org/batch/-/batch-0.5.3.tgz",
+                    "integrity": "sha1-PzQU84AyF0O/wQQvmoP/HVgk1GQ=",
+                    "dev": true
+                },
+                "camelcase": {
+                    "version": "3.0.0",
+                    "resolved": "https://registry.npmjs.org/camelcase/-/camelcase-3.0.0.tgz",
+                    "integrity": "sha1-MvxLn82vhF/N9+c7uXysImHwqwo=",
+                    "dev": true
+                },
+                "connect": {
+                    "version": "3.5.0",
+                    "resolved": "https://registry.npmjs.org/connect/-/connect-3.5.0.tgz",
+                    "integrity": "sha1-s1dSWgtMH1BZnNmD4dnv7qlncZg=",
+                    "dev": true,
+                    "requires": {
+                        "debug": "2.2.0",
+                        "finalhandler": "0.5.0",
+                        "parseurl": "1.3.2",
+                        "utils-merge": "1.0.0"
+                    }
+                },
+                "debug": {
+                    "version": "2.2.0",
+                    "resolved": "https://registry.npmjs.org/debug/-/debug-2.2.0.tgz",
+                    "integrity": "sha1-+HBX6ZWxofauaklgZkE3vFbwOdo=",
+                    "dev": true,
+                    "requires": {
+                        "ms": "0.7.1"
+       
```

Comment:
The package-lock.json is 14k long :D

Reply:
Why is it submitting package-lock?

Should I remove this?

Reply:
It generates it automatically if you have npm 5 and above. I think we should remove this for now until @gnomeontherun and @youdz decide on which lock file to use. yarn-lock or package-lock